options-manager.js

```diff
@@ -102,6 +98,14 @@ function buildConfig(opts) {
 		config.rules['babel/object-curly-spacing'] = [2, 'never'];
 	}
 
+	if (opts.rules) {
+		for (var attr in opts.rules) {
+			if (opts.rules.hasOwnProperty(attr)) {
+				config.rules[attr] = opts.rules[attr];
+			}
```

Comment:
Because of this https://github.com/sindresorhus/xo/commit/bdfccc6db0fd3c1f3fae81128ce3dd7a9074cb6c